src/main/java/org/jfrog/gradle/plugin/artifactory/extractor/GradleBuildInfoExtractor.java

```diff
@@ -37,6 +40,7 @@ public GradleBuildInfoExtractor(ArtifactoryClientConfiguration clientConf, List<
     public BuildInfo extract(Project rootProject) {
         BuildInfo buildInfo = createBuildInfoBuilder().build();
         PackageManagerUtils.collectEnvAndFilterProperties(clientConf, buildInfo);
+        removeResolutionProperties(buildInfo);
```

Comment:
Let's explain here the purpose of this removal.

Reply:
I elaborated more in the method's Javadoc following this comment.